app/views/planning_applications/additional_services/edit.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :page_title do %>
+  Pre-application services - <%= t("page_title") %>
+<% end %>
+
+<% add_parent_breadcrumb_link "Home", root_path %>
+<% add_parent_breadcrumb_link "Application", planning_application_path(@planning_application) %>
+
+<% content_for :title, "Pre-application services" %>
+
+<%= render(
+      partial: "shared/proposal_header",
+      locals: {heading: "Edit pre-application services"}
+    ) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @planning_application,
+          url: planning_application_additional_services_url(@planning_application) do |form| %>
+      <%= form.govuk_error_summary %>
+
+      <%= form.govuk_check_boxes_fieldset :additional_services,
+            legend: {text: "Select required services"} do %>
+            <% PreapplicationService::TYPES.each do |type| %>
```

Comment:
For now, we only have one subtype of AdditionalService so I'm going to leave this as it is, but when we have multiple types we'll need more logic here to determine which options to show. (No idea in advance if the multiple types will be mutually exclusive or whether showing options from more than one type will make sense, so not going to try to predict.)